geo_heatmap.py

```diff
@@ -154,66 +141,99 @@ def loadZIPData(self, file_name, date_range):
                 self.loadKMLData(read_file, date_range)
         else:
             raise ValueError("unsupported extension for {!r}: only .json and .kml supported"
-                .format(file_name))
+                             .format(file_name))
 
     def updateCoord(self, coords):
         self.coordinates[coords] += 1
         if self.coordinates[coords] > self.max_magnitude:
-            self.max_coordinates = coords
+            self.max_coordinates = coords[0], coords[1]
             self.max_magnitude = self.coordinates[coords]
 
-    def generateMap(self, settings):
-        """Generates the heatmap.
-        
-        Arguments:
-            settings {dict} -- The settings for the heatmap.
-        
-        Returns:
-            Map -- The Heatmap.
-        """
-        tiles = settings["tiles"]
-        zoom_start = settings["zoom_start"]
-        radius = settings["radius"]
-        blur = settings["blur"]
-        min_opacity = settings["min_opacity"]
-        max_zoom = settings["max_zoom"]
-        
+    def generateMap(self, tiles, map_zoom_start=6, heatmap_radius=7,
+                    heatmap_blur=4, heatmap_min_opacity=0.2,
+                    heatmap_max_zoom=4):
+

```

Comment:
Why are you removing the documentation and changing it back here? Make sure you are using the latest version